tools/check_release_assets_uploaded.py

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env python3
+import argparse
+import os
+import requests
+import sys
+import time
+
+def die(message):
+    print(message)
+    sys.exit(1)
+
+# get_assets_for_release returns a list of URL strings representing the assets uploaded to the release with the given
+# version tag
+# Example output:
+# ['https://github.com/buildbuddy-io/buildbuddy/releases/download/v2.0.0/executor-enterprise-darwin-arm64',
+# 'https://github.com/buildbuddy-io/buildbuddy/releases/download/v2.0.0/buildbuddy-enterprise-linux-amd64']
+def get_assets_for_release(version_tag):
+    github_token = os.environ.get('GITHUB_TOKEN')
+    query_url = "https://api.github.com/graphql"
```

Comment:
can we do the same thing using the REST API? we don't have as much in-house experience with graphql and it'd be easier to read/maintain as plain old JSON.